src/common/transformer_ctx.h

```diff
@@ -64,6 +64,7 @@ struct DecoderContext {
     int inputSeqLen;
     // For custom usage
     int reserved1;
+    int sequenceID;
```

Comment:
why do we need it here?

Reply:
Searchers don't know which sequenceID is executing, and need to transmit the sequenceID to Searchers.
If don't put it in DecoderContext, we could put it in the returned result in CommonDecoder.forward().

Reply:
Done